src/frontend/src/app/audit-log/audit-log.component.css

```diff
@@ -13,4 +13,12 @@
 
 thead {
   text-align: center;
+}
+
+input {
+  width: 193px;
+}
+
+select {
+  width: 195px;
```

Comment:
tested as responsive? (dev tools - mobile/ tablet devices) and real devices?

Reply:
Tested. It is responsive.